experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -19,6 +19,10 @@
 # TODO(bentheelder): verify that this works and move it into the images
 CACHE_HOST="http://bazel-cache.default:8080"
 
+package_version () {
+    dpkg-query --showformat='${Version}' --show $1
```

Comment:
This is correct for getting a package version but not sufficient for tracking what `$CC` actually points to if we install other versions etc. updating in a bit with something smarter based on intentionally breaking the cache with planter and a local instance of the remote cache.

Reply:
maybe `$CC --version`? contains spaces and strings, so needs quoting.

I'm also curious how stdlib includes are accounted for in caching.

Reply:
also curious: does the caching take into account the `--crosstool` and `--cpu` flags?